### PR TITLE
API-10943 added list_groups_properties

### PR DESCRIPTION
**Pull request by maciejjwojcik**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/management/changelog/index.mdx [hunk 1]
```diff
@@ -37,6 +37,10 @@ The developer preview version provides a preview of the upcoming changes to the
 - The [**List Agents**](/management/configuration-api/v3.5/#list-agents) and [**Get Agent**](/management/configuration-api/v3.5/#get-agent) methods now return agent's `account_id`.
 - Added new additional `last_logout` agent field for methods [**List Agents**](/management/configuration-api/v3.5/#list-agents) and [**Get Agent**](/management/configuration-api/v3.5/#get-agent).
 
+### Properties
+
+- There's a new request, [List Groups Properties](/management/configuration-api/v3.5/#list_group_properties).
```
Comment (kamilatokarska) [inline src/pages/management/changelog/index.mdx:42]: I believe the link should finish with `/v3.5#list-groups-properties' as this one does not work 🤔

Comment (maciejjwojcik) [reply src/pages/management/changelog/index.mdx]: I changed it to `[List Groups Properties](/management/configuration-api/v3.5/#list-groups-properties).` Note the slash after v3.5, it's not in your suggestion, but it's in every other link, so I _think_ it's the correct way to do this. If not, I can change it.

Comment (kamilatokarska) [reply src/pages/management/changelog/index.mdx]: I have checked on the preview, and it is working :)

**Review by kamilatokarska — APPROVED** [review]
:)

**Pull request merged**